init.lua

```diff
@@ -34,6 +34,9 @@ local disallowed = {
 }
 
 minetest.register_on_prejoinplayer(function(name, ip)
+	if exemptions[name] then return end
+	
+	-- Check for disallowed and unexemptioned names
```

Comment:
That's okay, is needed before the discovery of disallowed names allow the name set in minetest.conf as value name = ; if is username admin, owner, Admin, Owner.


I try add this before as first check, but cant login too with reason: "Sorry, someone else is already using this.  Please pick another name.

I must add this line also to function to case insensivity .. i do not known Why but after this , user admin can login to game.